src/mockStoreActionsAndGetters.js

```diff
@@ -10,23 +10,27 @@ function mockStoreActionsAndGetters ({
     const moduleValue = modulesDictionary[moduleKey]
     const isNamespaced = Object.keys(moduleValue).includes('namespaced')
 
-    Object.keys(moduleValue.actions).forEach(key => {
-      const mockActionKey = isNamespaced ? `${moduleKey}/${key}` : key
-      const mockFn = jestFn().mockResolvedValue({})
-      actionsDictionary[mockActionKey] = mockFn
-      moduleValue.actions[key] = mockFn
-    })
+    if (moduleValue.actions) {
+      Object.keys(moduleValue.actions).forEach(key => {
+        const mockActionKey = isNamespaced ? `${moduleKey}/${key}` : key
+        const mockFn = jestFn().mockResolvedValue({})
+        actionsDictionary[mockActionKey] = mockFn
```

Comment:
Could you rewrite this using _reduce_?

Reply:
I find that using `reduce` when the desired output is a map/dictionary object ends up being a bit awkward.

Reply:
I am in agreement with @bobmanary, using `reduce` in this use case will make code less readable. Here is a version using `reduce`

```javascript
Object.entries(moduleValue.actions).reduce((actionsDictionary, [key]) => {
      const mockActionKey = isNamespaced ? `${moduleKey}/${key}` : key
      const mockFn = jestFn().mockResolvedValue({})
      actionsDictionary[mockActionKey] = mockFn
      moduleValue.actions[key] = mockFn
      return actionsDictionary
    }, actionsDictionary)
```